content/release-10-10-0/announcements-10-10-0.md

```diff
@@ -62,4 +62,4 @@ for more information on the `GenericResponse` event.
 * Role-based access control will be introduced in the next release which will mandate assigning users to the Machine Learning Global Roles or User groups to enable them to use Cumulocity IoT Machine Learning.
 * The old versions of [Generate time series model](https://cumulocity.com/guides{{< 10-10-0 >}}/machine-learning/api-reference/#post--generate-time-series-model-using-time-series-data) `{{url}}/service/zementis/timeseries` API and [Get time series model generation status](https://cumulocity.com/guides{{< 10-10-0 >}}/machine-learning/api-reference/#get--get-status-of-generation-of-the-time-series-model) `{{url}}/service/zementis/timeseries/{{model_name}}/status` API which were deprecated in 10.7 release will be removed in the next release. However, the new versions `{{url}}/service/zementis/train/timeseries` and `{{url}}/service/zementis/train/timeseries/{{model_name}}/status` will still be in production.
 * The `applyToAllModels` query parameter used in [Apply model group](https://cumulocity.com/guides{{< 10-10-0 >}}/machine-learning/api-reference/#post---apply-pmml-model-group-to-multiple-records) and  [Apply model group and show details](https://cumulocity.com/guides{{< 10-10-0 >}}/machine-learning/api-reference/#post---apply-pmml-model-group-to-multiple-records-and-show-details) API was deprecated as part of 10.9 release and will be removed in the next release. This parameter is replaced by `applyAllModels`.
-* The `applyToAllModels` field of the [Job Configuration](https://cumulocity.com/guides{{< 10-10-0 >}}/machine-learning/api-reference/#domain-model) type was deprecated as part of 10.9 release and will be removed in the next release. This parameter is replaced by `applyAllModels`.
+* The `applyToAllModels` field of the [Job Configuration](https://cumulocity.com/guides{{< 10-10-0 >}}/machine-learning/api-reference/#jobconfiguration) type was deprecated as part of 10.9 release and will be removed in the next release. This parameter is replaced by `applyAllModels`.
```

Comment:
looks like there could be a "/" missing between "guides" and "{{< 10-10-0 >}}", but since all the URLs are like that, probably not? :)

Reply:
No, this is correct. The variable will be resolved to /10-10-0.